_includes/libraries-trouble.md

```diff
@@ -31,3 +31,59 @@ Making orders in bulk require the order be consistent for all pools.
 1. Click _Create Order_, and enter the information as for a single order.
 
 A new order will be create for each pool with the same information.
+
+## How do I change the targeted sequencing type on a library?
+Targeted sequencing is connected to the dilution since the same library can be used for multiple targeted sequencing panels.
+
+1. From _Tracking_, _Dilutions_, check the library dilutions that need to be changed.
+1. From the toolbar, click _Edit_.
+1. Select a new _Targeted Sequencing_ from the list.
+1. Click _Save_.
+
+## How can I add a new targeted sequencing type, kit, or anything else in drop-down menus?
+For targeted sequencing and indices, please email gsi@oicr.on.ca or file a JIRA ticket in GSI Common to get assistance from the MISO team.
```

Comment:
The "please email [...] GSI Common" will also be included in the plain sample tutorial, so maybe use the variable that's named something like `contact-admin`?